### PR TITLE
Issue 10 admin console

### PR DESCRIPTION
**Pull request by mariosk89**

### CODE REVIEW

File: src/main/java/uk/andrewgorton/digitalmarketplace/alerter/filters/AdminRequiredFilter.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     * @param userDAO : the DAO that provides access to the users table in the database
+     *
+     * @return boolean
+     * */
+    public static boolean isAdmin(HttpSession session, UserDAO userDAO)
+    {
+        Long userId = (Long) session.getAttribute("userid");
+        User currentUser = userDAO.findById(userId);
+
+        if (session.getAttribute("authenticated") == null || !((boolean) session.getAttribute("authenticated"))) {
```
Comment (Rossiar) [inline src/main/java/uk/andrewgorton/digitalmarketplace/alerter/filters/AdminRequiredFilter.java]: `if x return false else if y return false`? why not just `if x && y return false`

File: src/main/java/uk/andrewgorton/digitalmarketplace/alerter/resources/BidManagerResource.java [hunk 2]
```diff
@@ -24,6 +25,7 @@ public BidManagerResource(BidManagerDAO bidManagerDAO) {
     @GET
     @Timed
     @LoginRequired
+    @AdminRequired
```
Comment (Rossiar) [inline src/main/java/uk/andrewgorton/digitalmarketplace/alerter/resources/BidManagerResource.java]: Users should definitely be able to read (and probably also add/delete) bid managers without having to take it through an admin

File: src/main/resources/uk/andrewgorton/digitalmarketplace/alerter/views/homepage.ftl [hunk 3]
```diff
@@ -19,6 +19,19 @@
         <td><a href="alert/">Alerts</a></td>
         <td>Create or modify who gets notified when a new opportunity is detected</td>
     </tr>
+    <#if usersLocation??>
+        <tr>
+            <td><a href="${usersLocation}">Users</a></td>
+            <td>View, disable or delete existing users</td>
+        </tr>
+    </#if>
+    <#if bidManagersLocation??>
```
Comment (Rossiar) [inline src/main/resources/uk/andrewgorton/digitalmarketplace/alerter/views/homepage.ftl]: Not sure that we need to check for this as parameters passed to this template should not be `null`

Comment (mariosk89) [inline src/main/resources/uk/andrewgorton/digitalmarketplace/alerter/views/homepage.ftl]: Why shouldn't they be null?

**Pull request merged**